1-introduction/introduction_Task2.py

```diff
@@ -0,0 +1,11 @@
+numbers = [1, 2, '0', '300', -2.5, 'Dog', True, 0o1256, None]
+for n, i in enumerate(numbers):
+    try:
+        numbers[n] = int(i)
+    except TypeError:
```

Comment:
1. You can use

```python
except (TypeError, ValueError):
```

2. You will fail if input list is empty
3. It is bad practice to change the original variable if it is not necessary. Create a new list and add new values to it.

Reply:
I've refactored the whole logic - the try/except block has been removed
Please review